openembedded-core-mbl/meta/classes/mbl-uboot-sign.bbclass

```diff
@@ -12,6 +12,9 @@
 UBOOT_DTB_IMAGE ?= "u-boot-${MACHINE}-${PV}-${PR}.dtb"
 UBOOT_NODTB_IMAGE ?= "u-boot-nodtb-${MACHINE}-${PV}-${PR}.${UBOOT_SUFFIX}"
 
+# Default partitions to read the boot.scr (FIT image)
+UBOOT_DEFAULT_BOOT_PARTITION ?= "1"
```

Comment:
I'm not keen on this being added to a class called `mbl-uboot-sign` - it doesn't have anything to do with signing.

If you want to put this somewhere that applies to all of the U-Boot recipes then you could add it to `meta-mbl-bsp-common/recipes-bsp/u-boot/u-boot%.bbappend`

Reply:
ok. I'll put it there.